keyboards/plexus75/rules.mk

```
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# QMK Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

```suggestion
BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
```
I recommend having lite mode, so the users can have an easy reset mode.